pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -162,7 +162,6 @@ func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 
 	fts := feature.Features{
 		EnableVolumeCapacityPriority:                 utilFeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority),
-		EnableMinDomainsInPodTopologySpread:          utilFeature.DefaultFeatureGate.Enabled(features.MinDomainsInPodTopologySpread),
```

Comment:
Why was this featuregate deleted?

Reply:
as the page https://github.com/kubernetes/kubernetes/blob/release-1.30/pkg/scheduler/framework/plugins/feature/feature.go#L22 showing, EnableMinDomainsInPodTopologySpread has be deleted from struct feature.Features. So we must delete the above line.